extension/src/cli/CreateDocumentation.ts

```diff
@@ -1,75 +1,85 @@
 import * as Telemetry from "../Telemetry";
 import * as path from "path";
 import * as fs from "fs";
-
 import * as Documentation from "../Documentation";
 import * as CliSettingsLoader from "../Settings/CliSettingsLoader";
 import { logger, setLogger } from "../Logging/LogHelper";
 import { ConsoleLogger } from "../Logging/ConsoleLogger";
 
 setLogger(new ConsoleLogger());
 
+const functionName = "CreateDocumentation.js";
+
 const usage = `
 Usage:
-$> node CreateDocumentation.js <path-to-al-app-folder> <path-to-output-folder> [<path-to-workspace.code-workspace>] [<path-to-tooltip-file>]
+$> node ${functionName} <path-to-al-app-folder> <path-to-output-folder> [<path-to-workspace.code-workspace>] [<path-to-tooltip-file>]
 
 Example:
-$> node CreateDocumentation.js "C:\\git\\MyAppWorkspace\\App" "C:\\Docs\\MyApp\\reference" "C:\\git\\MyAppWorkspace\\MyApp.code-workspace" "C:\\Docs\\MyApp\\tooltips.md"
+$> node ${functionName} "C:\\git\\MyAppWorkspace\\App" "C:\\Docs\\MyApp\\reference" "C:\\git\\MyAppWorkspace\\MyApp.code-workspace" "C:\\Docs\\MyApp\\tooltips.md"
 `;
 
+interface Parameters {
+  workspaceFolderPath: string;
+  outputFolderPath: string;
+  workspaceFilePath: string | undefined;
+  tooltipDocsFilePath: string | undefined;
+}
+
+function getParameters(args: string[]): Parameters {
+  if (args.length < 4 || args.length > 6) {
+    logger.log(usage);
+    process.exit(1);
+  }
+  return {
+    workspaceFolderPath: args[2],
```

Comment:
This is on me, but the name on this just don't feel right. 
It should be `appFolderPath` or similar, don't you think? 

The same in RefreshXLF.js...